index.js

```diff
@@ -11,11 +12,41 @@ server.get('/payments', (req, res) => {
     res.status(200).send(payments);
 });
 
+server.get('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    if(payment){
+        res.status(200).send(payment);
+    }
+    else{
+        res.status(404).send({message: 'Wrong id'});
+    }
+});
+
+server.post('/payments', (req, res) => {
+    const payment = {...req.body};
+    Object.assign(payment, {
+        id: uuidv4(),
+        status: 'Pending'
+    });
+    payments.push(payment);
+    res.status(200).send(payment);
+});
+
+server.put('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    payment.description = req.body.description;
```

Comment:
do we want to validate the value of `req.body.description` what if it's undefined or an empty string?